src/Monolog/Processor/UidProcessor.php

```diff
@@ -43,4 +45,9 @@ public function getUid(): string
     {
         return $this->uid;
     }
+
+    public function restart(int $length = 7)
```

Comment:
I'd suggest to store the length as a property instead of adding in here

Reply:
agreed. None of the APIs are calling it with a value anyway, and we want it to be called with the original config.